# goal definition
---
# result definition
bool model_trained
---
# feedback definition
